.github/workflows/ci.yml

```
@@ -64,8 +64,30 @@ jobs:
+
+      - name: Verify OpenAPI schema is committed
+        run: git diff --exit-code apir/doc/openapi.yaml
+
       - name: Lint
         run: docker compose -f docker-compose.semaphore.yml run --no-deps --rm -T apir bin/lint
+
+  frontend-types:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)